src/apps/filters/h3.c

```diff
@@ -48,14 +48,19 @@ bool cellToLatLngCmd(int argc, char *argv[]) {
     Arg helpArg = ARG_HELP;
     DEFINE_CELL_ARG(cell, cellArg);
     Arg *args[] = {&cellToLatLngArg, &helpArg, &cellArg};
-    if (parseArgs(argc, argv, 3, args, &helpArg,
+    if (parseArgs(argc, argv, sizeof(args) / sizeof(Arg *), args, &helpArg,
                   "Convert an H3 cell to a latitude/longitude coordinate")) {
         return helpArg.found;
     }
     LatLng ll;
-    H3_EXPORT(cellToLatLng)(cell, &ll);
-    printf("%.10lf, %.10lf\n", H3_EXPORT(radsToDegs)(ll.lat),
-           H3_EXPORT(radsToDegs)(ll.lng));
+    H3Error err = H3_EXPORT(cellToLatLng)(cell, &ll);
+    if (err) {
+        return false;
+    }
+    // Using WKT formatting for the output. TODO: Add support for JSON
+    // formatting
```

Comment:
I'm curious about plans for output and input formats here. I assume for output we'd want a `-f` or `--format` param similar to `ogr2ogr`. It would be easy to go overboard here, but I think at a minimum:
- `csv`: `37.5012466151,-122.5003039349`
- `wkt`: `POINT(-122.5003039349 37.5012466151)`
- `geojson`: `{"type": "Point", "coordinates": [-122.5003039349 37.5012466151]}`

We might also support `json-array` (`[-122.5003039349 37.5012466151]`) or `json-object` (`{"lat":-122.5003039349, "lng": 37.5012466151}`) and maybe an `--ordering` param specifying `latlng` or `lnglat`?

The parallel question is what inputs we support - accepting these formats as inputs is nicely symmetrical, but more painful than `--lat=x --lng=y` or bare arguments.

Not saying we _should_ do any of this, but it might be nice to know where we plan to draw the line. I think a good yardstick should be how easy it is to pipe data from a tool like `ogr2ogr` to and from these CLI commands.